src/supermarket/app/controllers/api/v1_controller.rb

```diff
@@ -46,7 +46,7 @@ def error(body, status = 400)
   #
   def init_params
     @start = params.fetch(:start, 0).to_i
-    @items = [params.fetch(:items, 10).to_i, 100].min
+    @items = [params.fetch(:items, 10).to_i, item_limit].min
```

Comment:
There's a part of me that thinks that the API should respond with a 4xx error when an API request has an `item` value greater than the limit. Then the requestor would get a clear "no soup for you" message instead of wondering why their request did not return their desired number of items.